cms/envs/bok_choy.py

```diff
@@ -64,6 +64,9 @@
     (TEST_ROOT / "staticfiles" / "cms").abspath(),
 ]
 
+DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
```

Comment:
It appears that these are the same settings that are already in lms/envs/bok_choy.py. Therefore I think this change makes sense.

https://github.com/edx/edx-platform/blob/master/lms/envs/bok_choy.py#L67

FYI @jzoldak, in case you have any concerns.